code/model/SiteTree.php

```diff
@@ -804,10 +799,9 @@ public function NestedTitle($level = 2, $separator = " - ") {
 	 *
 	 * @uses DataObjectDecorator->can()
```

Comment:
Agreed. I would only use it where there are less-than-obvious side-effects, for example `@uses DataObject->delete()` on `DataList#remove`. But not in this instance.